### PR TITLE
OPS-5699 add IPS to payments-processor

### PR DESCRIPTION
**Pull request by barnesjake**

### CODE REVIEW

File: app/pp/model/wokitems/CdsOpsWorkItem.scala [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+case class CdsOpsWorkItem(
+    createdOn:       LocalDateTime,
+    availableUntil:  LocalDateTime,
+    taxType:         TaxType,
+    origin:          Origin,
+    reference:       String,
+    notificationCds: NotificationCds) extends WorkItemFields {
+
+  override def toString: String =
+    s"CdsWorkItem for chargeReference $reference ... {createdOn: $createdOn, availableUntil: $availableUntil, taxType: $taxType, origin: $origin, reference: $reference}"
```
Comment (andrewrichards1) [inline app/pp/model/wokitems/CdsOpsWorkItem.scala:34]: Minor .  You might want to include notifcationCDS

File: app/pp/connectors/CdsConnector.scala [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class CdsConnector @Inject() (httpClient: HttpClient, servicesConfig: ServicesConfig)(implicit ec: ExecutionContext) {
+
+  private val logger: Logger = Logger(this.getClass.getSimpleName)
+  private val serviceURL: String = s"${servicesConfig.baseUrl("cds")}/accounts/notifyimmediatepayment/v1"
+  private val authorizationToken: String = servicesConfig.getString("cds.authToken")
```
Comment (andrewrichards1) [inline app/pp/connectors/CdsConnector.scala:35]: Does not match config

**Review by andrewrichards1 — CHANGES_REQUESTED** [review]
Need do check the des config for bearer token

File: app/pp/connectors/CdsConnector.scala (hunk 2 above)
Comment (barnesjake) [reply app/pp/connectors/CdsConnector.scala:35]: Which bit?

Comment (barnesjake) [reply app/pp/connectors/CdsConnector.scala:35]: I've moved /accounts/notifyimmediatepayment/v1 into the callback method and simplified serviceUrl to just the cds base

Comment (barnesjake) [reply app/pp/connectors/CdsConnector.scala:35]: And put authToken in

cds {
        host = localhost
        port = 9975
        authToken = "cdsAuthToken"
    }

Comment (barnesjake) [reply app/pp/connectors/CdsConnector.scala:35]: sorry I'm with you, microservice.services...

**Comment by barnesjake** [comment]
test this please

**Review by andrewrichards1 — APPROVED** [review]

**Pull request merged**